lowranceusr.cc

```diff
@@ -894,18 +1884,75 @@ lowranceusr_route_hdr(const route_head* rte)
   if (global_opts.debug_level >= 1)
     printf(MYNAME " route_hdr: route name \"%s\" num_legs = %d\n",
            qPrintable(rte->rte_name), num_legs);
+}
+
+static void
+lowranceusr4_route_hdr(const route_head* rte)
+{
+  if (global_opts.debug_level >= 1) {
+    printf(MYNAME " writing route #%d (%s) with %d waypts\n",
+           route_uid, qPrintable(rte->rte_name), rte->rte_waypt_ct);
+  }
 
+  /* UID unit number */
+  if (opt_serialnum_i > 0)
+    gbfputint32(opt_serialnum_i, file_out);   // use option serial number if specified
+  else if (rte->fs != (format_specific_data *)NULL)
```

Comment:
nullptr instead of "(format_specific_data *) NULL" ?

Reply:
Adopted change.